crates/astria-conductor/local.env.example

```diff
@@ -33,3 +33,6 @@ ASTRIA_CONDUCTOR_SEQUENCER_URL="ws://127.0.0.1:26657"
 # true: skips empty blocks
 # false: executes empty blocks
 ASTRIA_CONDUCTOR_DISABLE_EMPTY_BLOCK_EXECUTION=false
+
+# the sequencer block height that the rollup genesis block was in
```

Comment:
```suggestion
# the sequencer block height that the rollup's first block was in
```

Reply:
the genesis block of the rollup is determined by the genesis state/file - the first block that uses the sequencer would be block 1